File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.py
class Solution:
    def reverseDegree(self, s: str) -> int:
        total_sum = 0
        for i, ch in enumerate(s):
            char_val = 26 - (ord(ch) - ord('a'))
            total_sum += char_val * (i + 1)
        return total_sum

File: 3498-reverse-degree-of-a-string/3498-reverse-degree-of-a-string.java
class Solution {
    public int reverseDegree(String s) {
        int total = 0;
        int n = s.length();

        for (int i = 0; i < n; i++) {
            int charVal = 26 - (s.charAt(i) - 'a');
            int pos = i + 1;
            total += pos * charVal;
        }

        return total;
    }
}